src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -1107,14 +1114,16 @@ internal RegexNode ScanBasicBackslash()
                 ch = RightChar();
             }
 
-            // Try to parse backreference: \<1> or \<cap>
+            // Try to parse backreference: \<1>
```

Comment:
the comment seemed right.  it was just referring to the if and the else section.